indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -74,6 +74,26 @@
             $reviewBox.find('.js-edit-review').trigger('click');
             $('body, html').animate({scrollTop: $reviewBox.offset().top}, 'fast');
             $reviewBox.find('.i-timeline-item-box').effect('highlight', {color: Palette.highlight}, 'slow');
+        }).on('click', '.js-toggle-details', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+
+            function toggleDetails($trigger) {
+                var target = $trigger.data('target');
+                var wasVisible = $trigger.hasClass('js-details-visible');
+                $trigger.closest('.track-review').find('.track-review-details .' + target).slideToggle('fast');
+                $trigger.toggleClass('js-details-hidden js-details-visible');
+                $trigger.text(wasVisible ? $trigger.data('show-text') : $trigger.data('hide-text'));
+            }
+
+            if ($this.hasClass('js-details-hidden')) {
+                $('.js-details-visible').each(function() {
+                    toggleDetails($(this));
+                });
+            }
+            toggleDetails($this);
+        }).on('click', '.js-heighlight-review', function() {
+            $($(this).attr('href') + ' .i-timeline-item-box').effect('highlight', 800);
```

Comment:
Isn't `'fast'` not fast enough? I think it would be better to stick to standard speeds than arbitrary speed values.